lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3383,48 +3384,47 @@ def test_permuations(self):
             with _set_mfe_flag(True):
                 # (preview=on, redirect=on)
                 # Global and Course Staff can see the link.
```

Comment:
The same set of changes is being made for all these assertions:
* just pass True or False in for is_global_staff, instead of a user whose is or isn't global staff.
* function was renamed from `show_courseware_mfe_link` to `courseware_mfe_is_advertised`.
* the course key is now the first argument instead of the third.